OutOfSchool/OutOfSchool.AuthCommon/Services/ProviderAdminService.cs

```diff
@@ -91,6 +91,7 @@ async Task<ResponseDto> CreateProviderAdminOperation(CreateProviderAdminDto crea
                 user.Role = nameof(Role.Provider).ToLower();
 
                 var result = await userManager.CreateAsync(user, password);
+                var errorMessages = result.ErrorMessages();
```

Comment:
(Code style) `errorMessages` is used in and relevant for only `if` clause. Please move it there. 
As a general rule of thumb - please create variables as close to their usage as possible.

Reply:
Oops... Fixed! Thank you!

Reply:
Thanks